etc/baryssh-client.conf

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I made them separate with fought that one can serve masked SSH service and connect to other masked SSH service, also. What do you think?